sdk/servicebus/azure-servicebus/swagger/generate_attributes_sequence_tuple.py

```diff
@@ -4,10 +4,10 @@
 # --------------------------------------------------------------------------------------------
 
 
-# This is a tool, not a part of the SDK code. Run it with Python 3.6+.
+# This is a tool, not a part of the SDK code. Run it with Python 3.7+.
 #
 # It iterates the _attribute_map of model classes and print the attributes of each class into a tuple.
-# `dict` in Python 3.6+ guarantees order when iterating over the dict so the attributes tuple
+# `dict` in Python 3.7+ guarantees order when iterating over the dict so the attributes tuple
```

Comment:
This comment is a clarification on the behavior of `dict`, and this behavior applies to Python 3.6 as well. For this reason, we don't need to make this update.